includes/compat/trait-wc-stripe-subscriptions.php

```
@@ -1267,22 +1267,33 @@ public function update_subscription_payment_method_from_order( $order, $payment_
 		$parent_order = wc_get_order( $order->get_parent_id() );
-		if ( WC_Stripe_Subscriptions_Helper::is_subscriptions_enabled()
-			&& $this->is_subscription( $order )
-			&& $parent_order
-			&& ! empty( WC_Stripe_Order_Helper::get_instance()->get_stripe_mandate_id( $parent_order ) ) ) {
-			$editable = false;
+		if ( ! $parent_order ) {
+			return $editable;
+		}
+
+		// Retrieve the payment method object from Stripe.
+		$payment_method_id = WC_Stripe_Order_Helper::get_instance()->get_stripe_source_id( $parent_order );
```

PHPMD (2.15.0)

[error] The variable $payment_method_id is not named in camelCase. (undefined)

(CamelCaseVariableName)